tools/test-lib/react-app/jest.setup.js

```diff
@@ -0,0 +1,4 @@
+// https://github.com/plotly/react-plotly.js/issues/115
```

Comment:
@rashidakanchwala this fixes the `window.URL.createObjectURL` is not a function issue. The other day we were adding it to the wrong place. It was failing inside this test for importing KedroViz in react-app, not at the top level tests.